meta/v1_scope.md

```diff
@@ -0,0 +1,14 @@
+# Initial-Version Scope
+
+The goal and scope of this initial version is to produce a usable rule book for *International Bible Quizzing* (IBQ) and district level quizzing that is *functionality equivalent* (FE) to the existing or current rule book. What this means is that if someone were to read the completed product of this project's initial version, they should render a quiz and meet that is indistinguishable from quizzes and meets at the IBQ level.
+
+Given the buggy and inconsistent of the current rule book, this is actually impossible because there are instances where the rule book itself contradicts itself and other instances where IBQ does not follow the rule book. Here's how we'll resolve these situations:
+
+- Where the rule book contradicts itself, we will write language in the new rule book such that the resulting quizzes and meets will match IBQ
+- Where IBQ does not follow the rule book, we will use the intended defintions from the current rule book
+
+## Rule Book Versus Best Practices
```

Comment:
I mean I like this, but makes me a small bit unhappy (more OCD-wise than anything). We define the requirements for a challenge in the Challenge section...but this feels like a blanket over the top.

Reply:
Discussed this with Scott offline. I think the content could be beefed up a bit to be more clear. I'll work on that.